src/components/__snapshots__/Image.test.js.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders correctly 1`] = `
+<Image
+  accessibilityLabel="Test Label"
+  resizeMode="contain"
+  source={
+    Object {
+      "height": 357,
+      "id": "3CU8n6O760GJ80Wr7r",
+      "revision": 1,
+      "uri": "//red-badger.com/G36Nw4N4Qb1HE305M0V.jpg",
```

Comment:
Hm, there might be a bug in `getImageDetails` that I overlooked earlier. Contentful gives us the uri without a scheme. I thought React Native requires a scheme, which is why we added "https:" in `getAssetSource` to the uri before. However I tested the build and it seems to work just fine. I am a bit confused. Does this just work now?!

Reply:
The code will add the `https` (the logic is in the image decoder). I just forgot to make the test data generator for ImageDetails add this.